bin/tests/common.rs

```diff
@@ -184,32 +185,43 @@ pub fn spawn_agent(settings: AgentSettings) -> Child {
         agent.env("LOGDNA_TAGS", tags);
     }
 
+    if let Some(journald_dirs) = settings.journald_dirs {
+        agent.env("LOGDNA_JOURNALD_PATHS", journald_dirs);
+    }
+
     agent.spawn().expect("Failed to start agent")
 }
 
+/// Blocks until a certain event referencing a file name is logged by the agent
+pub fn wait_for_file_event(event: &str, file_path: &PathBuf, reader: &mut dyn BufRead) -> String {
```

Comment:
💯

Reply:
The &PathBuf could probably be a Path, but there's very little difference

Reply:
Changed it!